podium-gateway/src/main/java/org/bbmri/podium/domain/enumeration/RequestType.java

```diff
@@ -0,0 +1,8 @@
+package org.bbmri.podium.domain.enumeration;
+
+/**
+ * The RequestType enumeration.
+ */
+public enum RequestType {
+    DATA,IMAGES,MATERIAL
```

Comment:
Not sure if this is needed (one biobank may have images, another may have material), but probably it should be a set of request types then? If you want both images and material?